doc/modules/model_evaluation.rst

```diff
@@ -6,6 +6,113 @@
 Metrics and scoring: quantifying the quality of predictions
 ===========================================================
 
+.. _which_scoring_function:
+
+Which scoring function should I use?
+====================================
+
+Before diving into the details of the many scores and metrics, we want
+to give some guidance, inspired by statistical decision theory, on the choice
+of **scoring functions** for **supervised learning**, see [Gneiting2009]_:
+
+- *Which scoring function should I use?*
+- *Which scoring function is a good one for my task?*
+
+In a nutshell, if the scoring function is given, e.g. in a kaggle competition
+or in a business context, use that one.
+If you are free to choose, it starts by considering the ultimate goal and application
+of the prediction. It is useful to distinguish two steps:
```

Comment:
🤔 It is a conceptual distinction that you *should* do in your mind. It is *not* something we baked into our API and often let to confusion (because `predict` hard-codes a decision threshold of 50% for binary classifiers). Only recently, we have begun to make that distinction more prominent, see https://scikit-learn.org/stable/modules/classification_threshold.html.